DataFormats/Scouting/interface/Run3ScoutingVertex.h

```diff
@@ -45,9 +54,12 @@ class Run3ScoutingVertex {
   float x() const { return x_; }
   float y() const { return y_; }
   float z() const { return z_; }
-  float zError() const { return zError_; }
   float xError() const { return xError_; }
   float yError() const { return yError_; }
+  float zError() const { return zError_; }
+  float xyCov() const { return xyCov_; }
+  float xzCov() const { return xyCov_; }
```

Comment:
```suggestion
  float xzCov() const { return xzCov_; }
```

Reply:
Done in [559a119](https://github.com/cms-sw/cmssw/pull/43758/commits/559a119fff075624ae95da320e667e015feb06a8)